src/app.js

```diff
@@ -122,14 +122,15 @@ export default function(app) {
   }
 
   function setElementData(element, name, value, oldValue) {
-    if (name === "key") {
-    } else if ((name = name.toLowerCase()) === "style") {
+    var attr = name.toLowerCase();
```

Comment:
I think we can skip this check altogether, since we only want to call toLowerCase once in line 133.